packages/ddp-client/common/livedata_connection.js

```diff
@@ -550,6 +550,37 @@ export class Connection {
     }
     return this.apply(name, args, callback);
   }
+  /**
+   * @memberOf Meteor
+   * @importFromPackage meteor
+   * @alias Meteor.callAsync
+   * @summary Invokes an async method passing any number of arguments.
+   * @locus Anywhere
+   * @param {String} name Name of method to invoke
+   * @param {EJSONable} [arg1,arg2...] Optional method arguments
+   * @returns {Promise}
+   */
+  async callAsync(name /* .. [arguments] .. callback */) {
+    const args = slice.call(arguments, 1);
+    if (args.length && typeof args[args.length - 1] === 'function') {
+      throw new Error(
+        "Meteor.callAsync() does not accept a callback. You should 'await' the result, or use .then()."
+      );
+    }
```

Comment:
I'm still thinking about it, but... Is there a need for this check? I know, it's "safe", but we don't really add them in other `*Async` methods. (Also, the comment in the function arguments could be updated.)

Reply:
We're going to keep it for now, as this won't affect the current apps. But we can revisit this later.